### DISCUSSION TITLE
Create a mac vm with T2 Security Chip and Apple Silicon?

### DISCUSSION THREAD

**Discussion by luisfscripcaru** [quickemu-project/quickemu#1464] [Quickemu]
Is there a way to create a mac with T2 Security Chip and Apple Silicon for enrolling into apple developer?
[Apple developer requirements](https://developer.apple.com/support/app-account/)

**Comment by lj3954** [comment]
Quickemu only supports x86_64 macOS. I believe Apple's ARM chips have custom instructions, making emulation of their ARM software very difficult. Performance for arm64 emulation on x86_64 devices is also very poor, and a GUI run through said emulation would be nearly unusable.
